### ISSUE TITLE
<PCPG (Chapter 36) Form Updates> from <Benefits - Non-Disability Experience Team>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#77805]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Benefits - Non-Disability Experience 
- **OCTO product owner:** Emily Theis
- **Product name:** Personalized Career Planning and Guidance Program (PCPG) aka Chapter 36 (25-8832)
- **Product manager:** Laura Steele
- **Slack channel:** #benefits-non-disability
- **Dedicated content writer on your team (if you have one):** none
- **Dedicated a11y specialist on your team (if you have one):** none
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [x] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

The current online Personalized Career Planning and Guidance Program (PCPG) aka Chapter 36 (25-8832) form is using an outdated version of the paper form. It's missing new fields and generating an old version of the form pdf. It also contains some accessibility issues that need to be addressed.

Desired Outcome: Veterans submitting online form 25-8832 have a positive experience and can trust that their submissions include all the required information to ensure efficient processing.

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
@fiorella-io has drafted the initial designs for the form updates in Figma, and we are reviewing those designs with VBA SMEs and our OCTO stakeholders.

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** [76364](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76364)

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Form Flow in Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691434304975/3f0c405c820c184d21393b3bbd76324a5ee7c677?sender=u934f98f179a86c76e6ee9592)
- [Figma Designs](https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-PCPG-2024-Updates?type=design&node-id=82%3A34295&mode=design&t=lfe6XyTm0esTbGp1-1)
- [VR&E Info Page](https://www.va.gov/careers-employment/education-and-career-counseling/)
- [Form update initiative brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/education-career-counseling/Initiative-Brief-PCPG-Form-Updates.md)
- [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/education-careers/education-career-counseling/README.md)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

**Comment by steele-lm** [comment]
cc @fiorella-io and @jstrothman

**Comment by jennymayoco** [comment]
Hi @steele-lm we've updated the best bet for form number 27-8832 so it's now the top result. We've also vetted the URL change and have passed on to Public Websites to implement the redirect, which can be tracked in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78118). Other than that, we don't have any additional IA feedback.

Just to confirm, is the PCPG team implementing the redirect for the PDF?

**Comment by steele-lm** [comment]
@jennymayoco The redirect for the pdf needs to be implemented by the VBA (Fran Cross), correct?

**Comment by jennymayoco** [comment]
@steele-lm Fran Cross is part of the Public Websites team. They're not able to implement redirects for PDFs and so Jill Adams had led the conversation with VBA last time. Both Jill and Dave Conlon are OOO at the moment so I can reach out to them to see if we will be able to redirect the PDF with the support of VBA again. Can you submit a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request) for that? That way Public Websites team can track the work there.

Will you be implementing the redirect for the form URL? A redirect ticket won't be needed if PCPG team is implementing the redirect, but if CAIA / Public Websites is implementing it, then we'll need a redirect request for that as well and will need to coordinate the URL change with the redirect to minimize any 404s.

**Comment by steele-lm** [comment]
@jennymayoco Thanks for clarifying! I submitted a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78560) for the pdf. In the ticket, I also included a link to the pdf redirect request from October 2023 in case PW needs to redirect the redirects 🙃. I have a [redirect request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78077) in with the PCPG team for the form url.

**Comment by katherine-fung** [comment]
Hi @fiorella-io! I've finished leaving content recommendations in the Figma file for the Ch 36 form. https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-PCPG-2024-Updates?type=design&mode=design&t=pjMexekV3QqpNANV-0

I asked VA folks: 
- are people on active duty always referred to as "service members"? 
- can people not on active duty be "Veterans or service members"? 

I didn't get a clear answer. But I made changes to get rid of all the conditional components except on this screen: https://www.figma.com/file/DEnVQXRrNvwJ3UkKkYYJ1Q/Ch-36-PCPG-2024-Updates?type=design&node-id=425-12234&mode=design&t=9wMnnTCBSheOSmdL-0 

When you talk to Ch 36 SMEs, do you think you can ask them to confirm whether the phrasing in my suggestion is correct? 

> <if you're a spouse or child applying for Ch 36 through someone NOT on active duty> Has the Veteran or service member completed other service periods? (*Required)
> 
> <if you're a spouse or child applying for Ch 36 through someone on active duty> Has the service member completed other service periods? (*Required)

Otherwise, I'm done reviewing this design. Thanks!
